xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -49,7 +49,7 @@ namespace XBMCAddon
       if (!label2.empty())
         item->SetLabel2( label2 );
       if (!iconImage.empty())
-        item->SetIconImage( iconImage );
+        item->SetArt("icon", iconImage );
```

Comment:
remove the space before `)`

Reply:
Line is removed by a subsequent commit

Reply:
The format should be fixed in the commit changing the line, even if it's removed in a later commit.

Reply:
Truly important. Done.

Reply:
Long live the bureaucracy. ;-)